meshparty/trimesh_vtk.py

```diff
@@ -482,14 +482,18 @@ def point_cloud_actor(xyz,
         mapper.SelectColorArray('colors')
     
     actor = vtk.vtkActor()
-    #mapper.ScalarVisibilityOff()
     actor.SetMapper(mapper)
     return actor
 
 
-def linked_point_actor(vertices_a, inds_a,
-                       vertices_b, inds_b,
+def linked_point_actor(vertices_a, vertices_b,
```

Comment:
this is technically a breaking change, but I don't think this is used that much right now so lets keep it.

Reply:
Agreed on both counts.